frontend/apps/reader/modules/readerkoptlistener.lua

```diff
@@ -16,15 +16,17 @@ end
 
 function ReaderKoptListener:onReadSettings(config)
     -- normal zoom mode is zoom mode used in non-reflow mode.
-    local normal_zoom_mode = config:readSetting("normal_zoom_mode") or
-                    G_reader_settings:readSetting("zoom_mode") or "page"
+    local normal_zoom_mode = config:readSetting("normal_zoom_mode")
+                          or G_reader_settings:readSetting("zoom_mode")
+                          or "page"
     normal_zoom_mode = util.arrayContains(ReaderZooming.available_zoom_modes, normal_zoom_mode)
-                    and normal_zoom_mode
+                   and normal_zoom_mode
                     or "page"
```

Comment:
Seems it should either be this or the one above with its space there included?
```suggestion
                   or "page"
```

Reply:
I tend to prefer aligning variables, yeah ;).

(Especially for an assignment).

Reply:
(Unless I'm getting cross-eyed, it's aligned on my end).

Reply:
Carry on then.